### PR TITLE
Add support django config detection for different versions (#170)

### LINKED ISSUE #170: RemovedInDjango41Warning due to default_app_config
**Describe the bug**

When starting a django application using this application on django 3.2, it generates this warning:

```
../usr/local/lib/python3.9/site-packages/django/apps/registry.py:91
  /usr/local/lib/python3.9/site-packages/django/apps/registry.py:91: RemovedInDjango41Warning: 'rest_framework_api_key' defines default_app_config = 'rest_framework_api_key.apps.RestFrameworkApiKeyConfig'. Django now detects this configuration automatically. You can remove default_app_config.
    app_config = AppConfig.create(entry)
```

**Desktop (please complete the following information):**
 - OS: Linux
 - Version: 2.0.0
 - Python Version: 3.9.2
 - Django Version: 3.2
 - DRF Version: 3.12.4

**Additional context**

See this for how the django_extensions handled this: https://github.com/django-extensions/django-extensions/pull/1654/files

### PR DESCRIPTION
**Pull request by syth0le**
Closes #170 
Fix problem with warnings that occurs after running pytests in any project with django version >= 3.2

### CODE REVIEW

**Review by florimondmanca — COMMENTED** [review]
Thanks! This is lookng good. I'm updating Django versions in #189 so we properly test this on < 3.2 (via `2.2.*`) and 3.2+ (via `3.2.*`) on CI.

**Comment by syth0le** [comment]
Great! That's would be a nice idea to test it on different django versions on CI

**Comment by florimondmanca** [comment]
@syth0le The versions updates got merged, feel free to rebase to see how this behaves against 2.2 and 3.2 :-)

**Comment by syth0le** [comment]
I think that all checks has been passed, but to be honest i dont know what to do with tests coverage (for 2 files). is it normal? 
Or do you need to more than 95 percent of tests coverage for each file?

File: .gitignore [hunk 1]
```diff
@@ -105,3 +105,5 @@ venv.bak/
 
 # databases
 *.sqlite3
+
+.idea/
```
Comment (florimondmanca) [inline .gitignore:109]: I found recently that you can add a global `.gitignore` config to your machine like this:

```bash
touch ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```

Helps prevent conflicts with developer dotfiles/dotdirs like these :) No problem including this though.

**Review by florimondmanca — APPROVED** [review]
Thanks!

**Comment by florimondmanca** [comment]
@syth0le I think codecov requires not reducing the coverage by default. We do have a code branch that's not run here (the one that's for < 3.2), and so the coverage decreased slightly. Not a problem, we can merge. I might look at if there's a way to have Codecov _not_ fail a build for this reason…

**Pull request merged**